recipes-bsp/tegra-binaries/tegra-libraries-gbm_32.6.1.bb

```diff
@@ -17,6 +17,13 @@ TEGRA_LIBRARIES_TO_INSTALL = "\
     tegra/libnvgbm.so \
 "
 
+do_install:append() {
+    patchelf --set-soname libnvgbm.so.1 ${D}${libdir}/libnvgbm.so
+}
+
+EXCLUDE_FROM_SHLIBS = "1"
+INSANE_SKIP:${PN} += "file-rdeps"
```

Comment:
Since you're setting the SONAME, you shouldn't need these settings - with the unique SONAME, the shlibs processing should "just work".  Worked OK for me in a test buid.